aspnetcore/tutorials/grpc/grpc-client.md

```diff
@@ -0,0 +1,224 @@
+
+In this tutorial, you:
+
+> [!div class="checklist"]
+> * Create a gRPC client.
+> * Run the service against the gRPC Greeter service created in the previous tutorial.
+> * Examine the project files.
+
+[!INCLUDE[](~/includes/net-core-prereqs-all-3.0.md)]
+
+## Create a .NET console application
+
+# [Visual Studio](#tab/visual-studio)
+
+Follow the instructions [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio) to create a console app with the name GrpcGreeterClient.
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+Follow the instructions [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/with-visual-studio-code) to create a console app with the name GrpcGreeterClient.
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+Follow the instructions [here](https://docs.microsoft.com/en-us/dotnet/core/tutorials/using-on-mac-vs-full-solution) to create a console app with the name GrpcGreeterClient.
+
+<!-- End of VS tabs -->
+
+---
+
+## Add required packages
+
+The following packages need to be added the gRPC client project:
+
+* [Grpc.Core](https://www.nuget.org/packages/Grpc.Core) which contains the C# API for the C-core client.
+* [Google.Protobuf](https://www.nuget.org/packages/Google.Protobuf/) which contains protobuf message APIs for C#.
+* [Grpc.Tools](https://www.nuget.org/packages/Grpc.Tools/) which contains C# tooling support for protobuf files. The tooling package isn't required at runtime, so the dependency is marked with `PrivateAssets="All"`.
+
+Packages can be added with the following approaches:
+
+### [Visual Studio](#tab/visual-studio)
+
+* From the **Package Manager Console** window:
+  * Go to **View** > **Other Windows** > **Package Manager Console**
+  * Navigate to the directory in which the *GrpcGreeterClient.csproj* file exists
+  * Execute the following command:
+
+    ```powershell
+    Install-Package Grpc.Core
+    ```
+
+  * Repeat for Google.Protobuf and Grpc.Tools
+
+
+* From the **Manage NuGet Packages** dialog:
+  * Right-click the project in **Solution Explorer** > **Manage NuGet Packages**
+  * Set the **Package source** to "nuget.org"
+  * Enter "Grpc.Core" in the search box
+  * Select the "Grpc.Core" package from the **Browse** tab and click **Install**
+  * Repeat for Google.Protobuf and Grpc.Tools
+
+### [Visual Studio for Mac](#tab/visual-studio-mac)
+
+* Right-click the *Packages* folder in **Solution Pad** > **Add Packages...**
+* Set the **Add Packages** window's **Source** drop-down to "nuget.org"
+* Enter "Grpc.Core" in the search box
+* Select the "Grpc.Core" package from the results pane and click **Add Package**
+* Repeat for Google.Protobuf and Grpc.Tools
+
+### [Visual Studio Code](#tab/visual-studio-code)
+
+Run the following command from the **Integrated Terminal**:
+
+```console
+dotnet add TodoApi.csproj package Grpc.Core
+```
+
+Repeat for Google.Protobuf and Grpc.Tools
+
+### [.NET Core CLI](#tab/netcore-cli)
```

Comment:
Selecting the .NET Core CLI tab doesn't work correctly. Its content appears below other content.

Could it be because its not an option ".NET Core CLI" is not an option in other tabs on this page?

Reply:
Yes probably, gonna remove this section for now.